docs/spec/relayer-old/README.md

```diff
@@ -0,0 +1,26 @@
+A TLA+ specification of the Relayer algorithm.
```

Comment:
Do we want to keep the old version of ICS18 in the repository? If yes, then I suggest we call it `v1` and locate it inside `docs/spec/relayer/` so we'll end up with a structure as follows:

```
docs/spec/relayer:
  --- v1:
         |-- ChannelHandlers.tla
         |-- Environment.tla 
         |-- README.md
         ....
  --- v2
         |-- ChannelHandlers.tla 
         |-- Chain.tla
         ....
```

I'm not a fan of calling anything "old" because that's relative (so the "refactored" new spec might become "old" at some point).

Reply:
We will not keep it after this PR is merged. It is just there as a stable version.